src/lib/time.js

```diff
@@ -270,25 +275,42 @@ var $builtinmodule = function (name) {
     %Z  Time zone name (no characters if no time zone exists).   
     %%  A literal '%' character.         
     */
-    // accept2dyear
-    mod.accept2year = function accept2year() {
-        throw new Sk.builtin.NotImplementedError("accept2year is not implemented")
-    };
 
-    // strftime()
-    mod.strftime = function strftime() {
-        throw new Sk.builtin.NotImplementedError("strftime is not implemented")
-    };
+    function strftime_f(format, t) {
+        throw new NotImplementedError("time.strftime() is not yet implemented");
+        Sk.builtin.pyCheckArgs("strftime", arguments, 1, 2);
```

Comment:
It is, but he plans to add the code in another PR so I think we can remove it from here. I am wondering anyways why the closure compiler, jshint or jscs does not complain about unreachable code.